app/controllers/application_controller.rb

```diff
@@ -6,21 +6,37 @@ def not_found
     render json: { error: 'not_found' }
```

Comment:
I haven't been able to get this PR to work with the leaderboard... after a short stint of bug searching I haven't been able to figure out the problem - do you have any suggestions Matt?

![image](https://github.com/openflighthpc/carbon-leaderboard/assets/102584263/aaf553f3-27f6-458f-a51d-cb8329b86f4f)

Reply:
That's one on my list to fix - it's not that the ENTIRE leaderboard is broken, just that the headers don't load properly if there aren't any entries. You should find that if you use the carbon client to send something to the leaderboard that it gets displayed properly

Reply:
This is fixed as of the latest version of this branch.